.rubocop_new.yml

```diff
@@ -0,0 +1,192 @@
+  Enabled: false
+Lint/TrailingCommaInAttributeDeclaration: # (new in 0.90)
+  Enabled: false
+Lint/UnreachableLoop: # (new in 0.89)
+  Enabled: false
+Lint/UselessMethodDefinition: # (new in 0.90)
+  Enabled: false
+Lint/UselessTimes: # (new in 0.91)
+  Enabled: false
+Lint/UriRegexp:
+  Enabled: false
+Style/AccessorGrouping: # (new in 0.87)
+  Enabled: false
+Style/ArrayCoercion: # (new in 0.88)
+  Enabled: false
+Style/BisectedAttrAccessor: # (new in 0.87)
+  Enabled: false
+Style/CaseLikeIf: # (new in 0.88)
+  Enabled: false
+Style/CombinableLoops: # (new in 0.90)
+  Enabled: false
+Style/ExplicitBlockArgument: # (new in 0.89)
+  Enabled: false
+Style/ExponentialNotation: # (new in 0.82)
+  Enabled: false
+Style/ExpandPathArguments:
+  Enabled: false
+Style/GlobalStdStream: # (new in 0.89)
+  Enabled: false
+Style/HashAsLastArrayItem: # (new in 0.88)
+  Enabled: false
+Style/HashLikeCase: # (new in 0.88)
+  Enabled: false
+Style/KeywordParametersOrder: # (new in 0.90)
+  Enabled: false
+Style/OptionalBooleanParameter: # (new in 0.89)
+  Enabled: false
+Style/RedundantAssignment: # (new in 0.87)
+  Enabled: false
+Style/RedundantFetchBlock: # (new in 0.86)
+  Enabled: false
+Style/RedundantFileExtensionInRequire: # (new in 0.88)
+  Enabled: false
+Style/RedundantRegexpCharacterClass: # (new in 0.85)
+  Enabled: false
+Style/RedundantRegexpEscape: # (new in 0.85)
+  Enabled: false
+Style/RedundantSelfAssignment: # (new in 0.90)
+  Enabled: false
+Style/SingleArgumentDig: # (new in 0.89)
+  Enabled: false
+Style/SlicingWithRange: # (new in 0.83)
+  Enabled: false
+Style/SoleNestedConditional: # (new in 0.89)
+  Enabled: false
+Style/StringConcatenation: # (new in 0.89)
+  Enabled: false
+Performance/AncestorsInclude: # (new in 1.7)
+  Enabled: false
+Performance/BigDecimalWithNumericArgument: # (new in 1.7)
+  Enabled: false
+Performance/DeletePrefix:
+  Enabled: false
+Performance/RedundantSortBlock: # (new in 1.7)
+  Enabled: false
+Performance/RedundantStringChars: # (new in 1.7)
+  Enabled: false
+Performance/ReverseFirst: # (new in 1.7)
+  Enabled: false
+Performance/SortReverse: # (new in 1.7)
+  Enabled: false
+Performance/Squeeze: # (new in 1.7)
+  Enabled: false
+Performance/StringInclude: # (new in 1.7)
+  Enabled: false
+Performance/CollectionLiteralInLoop:
+  Enabled: false
+Performance/Sum: # (new in 1.8)
+  Enabled: false
+Rails/ActiveRecordCallbacksOrder: # (new in 2.7)
+  Enabled: false
+Rails/AfterCommitOverride: # (new in 2.8)
+  Enabled: false
+Rails/ContentTag:
+  Enabled: false
+Rails/FindById: # (new in 2.7)
+  Enabled: false
+Rails/Inquiry: # (new in 2.7)
+  Enabled: false
+Rails/MailerName: # (new in 2.7)
+  Enabled: false
+Rails/MatchRoute: # (new in 2.7)
+  Enabled: false
+Rails/NegateInclude: # (new in 2.7)
+  Enabled: false
+Rails/Pluck: # (new in 2.7)
+  Enabled: false
+Rails/PluckInWhere: # (new in 2.7)
+  Enabled: false
+Rails/UniqueValidationWithoutIndex:
+  Enabled: false
```

Comment:
This one (`UniqueValidationWithoutIndex`) reveals we have number of validations without unique indices.